### DISCUSSION TITLE
How to use this python implementation？

### DISCUSSION THREAD

**Discussion by LaserLV52** [lassoan/pyigtl#15] [Q&A]
I am now planning to use 3Dslicer to reconstruct the offline ultrasound data, where each image corresponds to a 6-DOF coordinate, which is exactly the same as the two functions **pyigtl.PositionMessage** and **pyigtl.ImageMessage** in this code. This should be the effect I want to achieve.

But now the question is, how do I configure 3Dslicer to receive data from python? I've seen some slicerIGT tutorials in slicerigt.org and it seems like to fetch data from outside you have to use plus server and prepare a configuration file (.xml) for that.

I am now confused with how to write this configuration file right , any good idea? 

Very appreciate！

**Comment by DylanM-ihu** [comment]
Hello,

I am in a similar situation. Did you find how to receive data from Python with 3D Slicer?

Thanks a lot!

**Reply by LaserLV52** [reply]
Not yet. Seems like few people use this implementation even few poeple study in this topic. I also ask the same question in 3d sclicer forum, no one reply to me either.
